### PR TITLE
Add note about final classes

### PR DESCRIPTION
**Pull request by OskarStark**
Screenshot 2020-02-17 08 56 05

### CODE REVIEW

File: service_container/lazy_services.rst [hunk 1]
```diff
@@ -23,6 +23,10 @@ Configuring lazy services is one answer to this. With a lazy service, a
 just like the ``mailer``, except that the ``mailer`` isn't actually instantiated
 until you interact with the proxy in some way.
 
+.. note::
+
+    Lazy services does not work with final classes!
```
Comment (noniagriconomie) [inline service_container/lazy_services.rst]: I think the `!` is to much, do the documentation have a lot of this sentences?

File: service_container/lazy_services.rst [hunk 2]
```diff
@@ -23,6 +23,10 @@ Configuring lazy services is one answer to this. With a lazy service, a
 just like the ``mailer``, except that the ``mailer`` isn't actually instantiated
 until you interact with the proxy in some way.
 
+.. note::
```
Comment (HeahDude) [inline service_container/lazy_services.rst]: What about a `caution` instead?

**Review by ghost — APPROVED** [review]
√

**Review by HeahDude — APPROVED** [review]

File: service_container/lazy_services.rst [hunk 3]
```diff
@@ -23,6 +23,10 @@ Configuring lazy services is one answer to this. With a lazy service, a
 just like the ``mailer``, except that the ``mailer`` isn't actually instantiated
 until you interact with the proxy in some way.
 
+.. caution::
+
+    Lazy services does not work with final classes.
```
Comment (wouterj) [inline service_container/lazy_services.rst]:
```suggestion
    Lazy services do not work with final classes.
```

**Review by wouterj — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by HeahDude** [comment]
Closing this one as it's been merged already.

**Pull request closed without merging**